modules/administration-guide/examples/ref_che-supported-platforms-and-installation-methods.adoc

```diff
@@ -34,27 +31,27 @@ a|* `minikube`
 |OperatorHub
 |===
 
+WARNING: Setting up [users authentication](https://kubernetes.io/docs/reference/access-authn-authz/authentication/) is required for deploying {prod-short} on {kubernetes} infrastructures. For {ocp} no additional setup is needed.
```

Comment:
```suggestion
WARNING: Setting up [User's Authentication](https://kubernetes.io/docs/reference/access-authn-authz/authentication/) is required for deploying {prod-short} on {kubernetes} infrastructures. For {ocp} no additional setup is needed.
```

Reply:
@swatikarmarkar maybe `Users' Authentication` in this case?